src/Pages/userList.tsx

```diff
@@ -136,7 +128,7 @@ export const UserListPage = () => {
                       <DialogTitle>Block this user?</DialogTitle>
                       <DialogContent>
                         <DialogContentText>
-                          Do you really want you block this user?
+                          Do you really want to block this user?
```

Comment:
Chyba znowu coś się pokręciło z commitami i na tym PR są też moje commity na crud usera. Dasz radę to ogarnąć?

Reply:
Nie, poprawiłem błędy w twoim poprzednim PR.